src/partner-admin/partner-admin.controller.ts

```diff
@@ -41,4 +42,16 @@ export class PartnerAdminController {
   ): Promise<PartnerAdminEntity | unknown> {
     return this.partnerAdminService.createPartnerAdminUser(createPartnerAdminUserDto);
   }
+
+  @ApiBearerAuth('access-token')
+  @ApiOperation({ description: 'Update partner admin by id' })
+  @UseGuards(SuperAdminAuthGuard)
+  @Patch(':id')
+  @ApiBody({ type: UpdatePartnerAdminDto })
+  async updatePartnerAdminById(
+    @Param('id') partnerAdminId: string,
+    @Body() updatePartnerAdminDto: UpdatePartnerAdminDto,
+  ): Promise<boolean> {
```

Comment:
This wasn't specified on the ticket but it would be great to return the updated entity rather than a boolean. This gives a bit more flexibility to the client.